nixos/modules/services/continuous-integration/gitea-actions-runner.nix

```
@@ -211,7 +211,7 @@ in
             User = "gitea-runner";
             StateDirectory = "gitea-runner";
             WorkingDirectory = "-/var/lib/gitea-runner/${name}";
-            ExecStartPre = pkgs.writeShellScript "gitea-register-runner-${name}" ''
+            ExecStartPre = lib.singleton (pkgs.writeShellScript "gitea-register-runner-${name}" ''
```

Feel like this is more readable and shorter:

```suggestion
            ExecStartPre = [(pkgs.writeShellScript "gitea-register-runner-${name}" ''
```